package.json

```diff
@@ -69,8 +69,9 @@
   },
   "devDependencies": {
     "@types/debug": "^4.1.5",
+    "@types/got": "^9.6.11",
```

Comment:
`got` ships with its own type definitions.  Was there a reason you had to pull these in?

Reply:
Like I said before, that's my first try with TypeScript. Just thought it wouldn't compile like when I added the base https module. :) I'm removing it.